api/pom.xml

```diff
@@ -30,111 +30,113 @@
-    </licenses>
-
     <developers>
         <developer>
             <id>jakarta-ee4j-websocket</id>
             <name>Jakarta WebSocket Developers</name>
             <organization>Eclipse Foundation</organization>
             <email>websocket-dev@eclipse.org</email>
         </developer>
-        <developer>
-            <id>roman_grigoriadi</id>
-            <name>Roman Grigoriadi</name>
-            <organization>Oracle</organization>
-            <roles>
-                <role>developer</role>
-            </roles>
-        </developer>
     </developers>
 
     <contributors>
-        <contributor>
-            <name>Jakarta WebSocket Contributors</name>
-            <email>websocket-dev@eclipse.org</email>
-            <url>https://github.com/eclipse-ee4j/websocket-api/graphs/contributors</url>
-        </contributor>
-        <contributor>
-            <name>Jitendra Kotamraju</name>
-            <url>http://weblogs.java.net/blog/jitu</url>
-        </contributor>
-        <contributor>
-            <name>Danny Coward</name>
-            <organization>Oracle</organization>
-            <roles>
-                <role>developer</role>
-            </roles>
-        </contributor>
-        <contributor>
-            <name>Stepan Kopriva</name>
-        </contributor>
-        <contributor>
-            <name>Pavel Bucek</name>
-        </contributor>
+       <contributor>
+           <name>Jakarta WebSocket Contributors</name>
+           <email>websocket-dev@eclipse.org</email>
+           <url>https://github.com/eclipse-ee4j/websocket-api/graphs/contributors</url>
+       </contributor>
     </contributors>
 
+    <mailingLists>
+        <mailingList>
+            <name>WebSocket dev mailing list</name>
+            <post>websocket-dev@eclipse.org</post>
+            <subscribe>https://dev.eclipse.org/mailman/listinfo/websocket-dev</subscribe>
+            <unsubscribe>https://dev.eclipse.org/mailman/listinfo/websocket-dev</unsubscribe>
+            <archive>https://dev.eclipse.org/mhonarc/lists/websocket-dev</archive>
+        </mailingList>
+    </mailingLists>
+
+    <scm>
+        <connection>scm:git:https://github.com/eclipse-ee4j/websocket-api.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com/eclipse-ee4j/websocket-api.git</developerConnection>
+        <url>https://github.com/eclipse-ee4j/websocket-api</url>
+        <tag>HEAD</tag>
+    </scm>
+
+    <issueManagement>
+        <system>github</system>
+        <url>https://github.com/eclipse-ee4j/websocket-api/issues</url>
+    </issueManagement>
+
     <properties>
-        <api_package>jakarta.websocket</api_package>
-        <spec_version>2.0</spec_version>
-        <!--<spec_build>02</spec_build>-->
-        <new_spec_version>2.0</new_spec_version>
-        <spec_impl_version>2.0.0</spec_impl_version>
-        <packages.export>jakarta.websocket.*</packages.export>
+        <!-- Make sure the two versions are in sync with the maven version -->
```

Comment:
I personally think it's more clear if these are directly in the bundle plug-in. We don't re-use them, and we don't need to override them from the command line.

That said, not a blocker, just a preference.